clients/src/main/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManager.java

```diff
@@ -483,6 +483,23 @@ public void resetTimer() {
             this.heartbeatTimer.reset(heartbeatIntervalMs);
         }
 
+        @Override
+        public String toStringBase() {
+            return super.toStringBase() +
+                    ", heartbeatTimer=" + heartbeatTimer +
+                    ", heartbeatIntervalMs=" + heartbeatIntervalMs;
+        }
+
+        // Visible for testing
```

Comment:
i don't think we need the `heartbeatTimer()` and `heartbeatIntervalMs()` here.  see the comment below.